python/GafferSceneUITest/TranslateToolTest.py

```diff
@@ -1015,5 +1021,29 @@ def testTransformInEditScopeButEditScopeOff( self ) :
 		self.assertEqual( tool.handlesTransform(), imath.M44f().translate( imath.V3f( 1, 1, 0 ) ) )
 		self.assertEqual( transformEdit.translate.getValue(), imath.V3f( 1, 1, 0 ) )
 
+	def testNonEditableSelections( self ) :
```

Comment:
Do we also want to add `tool.selectionEditable()` checks in https://github.com/GafferHQ/gaffer/blob/9a66d3f5ac5669768fe1f2594ccb792879baac7b/python/GafferSceneUITest/TransformToolTest.py#L143

Reply:
No, that test is working with `TransformTool.Selection` directly, so there isn't a TransformTool to call `selectionEditable()` on.

Reply:
Ahh yes, sorry. In that case, should we have a test where the selection shouldn't be editable?

Reply:
Sorry, right you are - had missed that.